15_2_Mining_with_Regtest.md

```diff
@@ -0,0 +1,33 @@
+# 15.2: Mining the Regtest
+
+> **NOTE:** This is a draft in progress, so that I can get some feedback from early reviewers. It is not yet ready for learning.
+
+This document explains how to generate (i.e. mine) blocks using a Regtest (Regression Test) blockchain.
+To generate the Genesis block and the next blocks on a new blockchain requires very minimal proof-of-work, due to the low difficulty and that it follows the Testnet rules.
+
+
+## Generating blocks
+
+You can generate/mine new blocks using the RPC method `generate`. This method is only available in the Regtest mode, using the following command:
```

Comment:
`generate` is technically available in all modes (I would call them networks not modes, actually), but it's not very likely to yield any blocks on mainnet or testnet due to the high difficulty.